pbspro/docs/guides/10-run-graphical-jobs.md

```diff
@@ -0,0 +1,35 @@
+# run-graphical-jobs(7) -- How to run graphical jobs
+
+## DESCRIPTION
+
+This guide will help you start your graphical applications in the correct way within your compute environment.
+
+## OVERVIEW
+
+Graphical applications should be run within the compute environment by submitting them to the job scheduler system. Running your applications in this way will provide them with access to scalable resources.
+
+## PROCEDURE
+
+Login to your graphical desktop and start a new terminal shell. At this point, you are logged in to a login node within the compute environment - applications should be submitted to the HPC job scheduler and not run on the login node directly.
+
+To gain an interactive session on one of the available cluster compute hosts - use the following example command:
+
+    `qsub -I`
+
+You may use the same syntax as the `sbatch` command to request additional resources for your interactive shell. For example, to request an interactive session with access to 32GB of RAM and a maximum runtime of 72 hours, use the following command:
```

Comment:
Is `sbatch` right here? I don't see any mention of it in the rest of the PBSPro docs...